printrun/gui/__init__.py

```diff
@@ -202,8 +204,7 @@ def createTabbedGui(self):
         self.Bind(wx.EVT_CLOSE, self.kill)
 
         # Custom buttons
-        if wx.VERSION > (2, 9): self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
-        else: self.cbuttonssizer = wx.GridBagSizer()
+        self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
         self.cbuttonssizer = wx.GridBagSizer()
         self.centerpanel = self.newPanel(page1panel2)
         self.centerpanel.SetSizer(self.cbuttonssizer)
```

Comment:
@kliment however, it is used here, so if I just remove the 2 lines above, I break it.

Reply:
Ah never mind, they're in separate functions. Either way, I think we can drop the GridBagSizer from both.

Reply:
Amended.